packages/angular/src/app/components/contact-new-user-form/contact-new-user-form.component.html

```diff
@@ -10,11 +10,10 @@
     </dxi-item>
   </dxi-item>
 
-  <dxi-item itemType="group">
-    <dxo-col-count-by-screen [xs]="1" [sm]="1" [md]="2" [lg]="2"></dxo-col-count-by-screen>
+  <dxi-item itemType="group" [colCountByScreen]="{xs: (screen.smallMobileScreenChanged | async) ? 1 : 2, sm: 2, md: 2, lg: 2}">
```

Comment:
Why do you use an object to set colCountByScreen option? Could we use nested component instead?

Reply:
It's component feature. It's impossible change only "xs" prop